charts/gardener-extension-admission-calico/charts/application/templates/_helpers.tpl

```diff
@@ -0,0 +1,37 @@
+{{- define "name" -}}
+gardener-extension-admission-calico
+{{- end -}}
+
+{{- define "labels.app.key" -}}
+app.kubernetes.io/name
+{{- end -}}
+{{- define "labels.app.value" -}}
+{{ include "name" . }}
+{{- end -}}
+
+{{- define "labels" -}}
+{{ include "labels.app.key" . }}: {{ include "labels.app.value" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end -}}
+
+{{-  define "image" -}}
+  {{- if hasPrefix "sha256:" .Values.image.tag }}
+  {{- printf "%s@%s" .Values.image.repository .Values.image.tag }}
+  {{- else }}
+  {{- printf "%s:%s" .Values.image.repository .Values.image.tag }}
+  {{- end }}
+{{- end }}
+
+{{- define "deploymentversion" -}}
+apps/v1
+{{- end -}}
+
+{{- define "priorityclassversion" -}}
+{{- if semverCompare ">= 1.14-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
Do we wanna support such old k8s version (< 1.15)?

Reply:
No, I will remove it.
That being said I will open a separate pull request to clean the remaining calico code of this handling.

Reply:
Removed with new commit.

Reply:
Removed from other calico files with https://github.com/gardener/gardener-extension-networking-calico/pull/97